src/client/shader.cpp

```diff
@@ -45,7 +45,13 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include <GLES2/gl2.h>
 #endif
 #else
+#if defined(__APPLE__)
+#include <OpenGL/gl.h>
+#include <OpenGL/glu.h>
+#else
 #include <GL/gl.h>
+#include <GL/glu.h>
```

Comment:
This breaks several builds because glu.h is not present, we don't need that one either.
Does including `OpenGL/gl.h` suffice on macOS?

Reply:
Oh dang, sorry. Tried to keep it as minimal as possible. Yes I think so. Will verify ASAP

Reply:
Oh dang, sorry. Tried to keep it as minimal as possible. Yes I think so. Will verify ASAP